benchmarks/multiplatform/benchmarks/src/commonMain/kotlin/Args.kt

```diff
@@ -55,12 +59,28 @@ object Args {
 
     private fun String.decodeArg() = replace("%20", " ")
 
+    private fun reset() {
```

Comment:
Please comment this function providing reasoning when it is required

Reply:
fixed